lib/BaseTemplateController.js

```diff
@@ -25,14 +25,19 @@ module.exports = class BaseTemplateController extends CompositeController {
     super(params);
   }
 
-  async added() {
-    let fetchEnvs = new FetchEnvs(this);
-    let env = await fetchEnvs.get('spec');
+  concatTemplates() {
```

Comment:
is this necessary for your changes, or just an improvement on the code?

Reply:
i was trying to simplify something in the MTP PR. if this isn't added, then i would need to [duplicate the on line 42 of `bin/renderlocal`](https://github.com/razee-io/MustacheTemplate/pull/400/files#diff-fbc8c981c9d2b1021bc47de0cc645c278df3cd506d06903cfb80b0b5396e895aR42)